### ISSUE TITLE
Technique C36 missing content?

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#682]
[Allowing for text spacing override](https://www.w3.org/WAI/WCAG21/Techniques/css/C36) has a sentence which seems to have been truncated.
> Where text-content can wrap within its container authors should

**Comment by mbgower** [comment]
I looked through the history and it appears that sentence was never completed in any version in the repository. @WayneEDick do you have a draft copy that shows what was supposed to occur here? Whoever authored it should look for a version prior to or near Nov 16, 2018, as that is when the file appears to have been populated by Michael Cooper.

**Comment by lauracarlson** [comment]
Good catch @mbgower .

The truncated sentence is not in any of my local versions. @alastc may have it locally. He separated the original technique into 2 techniques in August 2018. Prior history is in Tech spacing override #386

**Comment by alastc** [comment]
I've had a look, but I think the original files are in a long deleted branch. I can't think what it would be, probably best to just delete it?

**Comment by lauracarlson** [comment]
@alastc wrote:
> I've had a look, but I think the original files are in a long deleted branch. I can't think what it would be, probably best to just delete it?

Yes. Deleting it is probably best then. Thanks for checking.

**Comment by alastc** [comment]
Ok, gone. If anyone thinks of what it should be, a quick PR would be in order. Thanks.
